src/khepri_fun.erl

```diff
@@ -317,9 +317,9 @@ to_standalone_fun1(Fun, Options) ->
     State0 = #state{fun_info = Info,
                     all_calls = #{{Module, Name, Arity} => true},
                     options = Options},
-    to_standalone_fun2(Fun, State0).
+    to_standalone_fun2_with_retry(Fun, State0).
 
--spec to_standalone_fun2(Fun, State) -> {StandaloneFun, State} when
+-spec to_standalone_fun2(Fun, State) -> {StandaloneFun, State} | undefined when
```

Comment:
Could you please rename this function to `to_standalone_fun2_locked()` and call the new function `to_standalone_fun2()`?

This is a common convention so we understand that `to_standalone_fun2()` and `to_standalone_fun2_locked()` are the same thing, and `to_standalone_fun2()` is responsible for wrapping the call to `to_standalone_fun2_locked()` in a lock.